lib/ts/recipeImplementation.ts

```diff
@@ -137,6 +145,62 @@ export default function RecipeImplementation(recipeImplInput: {
             }
 
             // we do not send an event here since it's triggered in setIdRefreshToken area.
+        },
+
+        getInvalidClaimsFromResponse: async function(input: {
+            response: AxiosResponse | Response;
+        }): Promise<ClaimValidationError[]> {
+            let body;
+            if ("body" in input.response) {
+                body = await input.response.clone().json();
+            } else {
+                body = input.response.data;
+            }
+
+            return body.claimValidationErrors;
+        },
+
+        getGlobalClaimValidators: async function(input: {
```

Comment:
this function is not actually being used anywhere? Shuoldn't we use it when validateClaim is called?